src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -8,6 +8,7 @@
 from __future__ import absolute_import as _
 
 import numpy as np
+import cv2
```

Comment:
I may be wrong, but I'm not sure we have taken a dependency on OpenCV. Is there a compelling reason for us to take a dependency on OpenCV here?

Reply:
Yes.
Tensorflow's resize function is not aligned with mps and Mxnet's,
which causes the inference regression as I show in the summary.
Numpy itself doesn't has resize method for image,
and that's why I'm using cv2's resize method now,
which is consistent between mps and mxnet.

Reply:
We would need to get legal approval to depend on cv2, and it's a huge dependency to pull in if it's needed only for image resizing.  We already resize images in many other places (see, e.g., the image_deep_feature_extraction code path).  

If there is a way to use PIL or one of our C++ image resizing utilities instead, we should do that.

Reply:
@hoytak Thanks for this suggestion!